src/SparkFun_u-blox_GNSS_Arduino_Library.cpp

```diff
@@ -8197,6 +8197,50 @@ bool SFE_UBLOX_GNSS::setupPowerMode(sfe_ublox_rxm_mode_e mode, uint16_t maxWait)
   return sendCommand(&packetCfg, maxWait);
 }
 
+
+// Position Accuracy
+
+// Change the Position Accuracy using UBX-CFG-NAV5
+// Value provided in meters
+bool SFE_UBLOX_GNSS::setNAV5PositionAccuracy(uint16_t meters, uint16_t maxWait)
+{
+  packetCfg.cls = UBX_CLASS_CFG;
+  packetCfg.id = UBX_CFG_NAV5;
+  packetCfg.len = 0;
+  packetCfg.startingSpot = 0;
+
+  // Ask module for the current navigation model settings. Loads into payloadCfg.
+  if (sendCommand(&packetCfg, maxWait) != SFE_UBLOX_STATUS_DATA_RECEIVED) // We are expecting data and an ACK
+    return (false);
+
+  payloadCfg[0] = 0x10;            // mask: set only the posMark
+  payloadCfg[1] = 0x00;            // mask
+  payloadCfg[18] = meters; 
```

Comment:
Please change ```meters``` to ```metres``` ;-)

You need to modify this line. pAcc is little-endian. Something like:

```
payloadCfg[18] = metres & 0xFF;
payloadCfg[19] = metres >> 8;
```

Reply:
Thanks for providing this. I'm not that good with bits so i struggled!